tests/vnc/vnc_test.go

```diff
@@ -142,14 +144,47 @@ func checkSSHAccess() projects.ProcTimerFunc {
 			HostKeyCallback: ssh.InsecureIgnoreHostKey(),
 			Timeout:         defaults.DefaultRepeatTimeout,
 		}
-		_, err := ssh.Dial("tcp", fmt.Sprintf("%s:%d", externalIP, *sshPort), config)
+		conn, err := ssh.Dial("tcp", fmt.Sprintf("%s:%d", externalIP, *sshPort), config)
 		if err != nil {
 			return nil
 		}
+		if *doPanic {
+			session, _ := conn.NewSession()
+			go func() {
+				_ = session.Run("sudo su -c 'echo c > /proc/sysrq-trigger'") //we cannot get answer for this command
+				session.Close()
+			}()
+			return fmt.Errorf("kernel panic fired via SSH on %s:%d", externalIP, *sshPort)
+		}
 		return fmt.Errorf("SSH success. You can access it via SSH on %s:%d", externalIP, *sshPort)
 	}
 }
 
+//checkPanicInLog try to find panic in app log
```

Comment:
Can we do a general function Check (message) in log and call it with string param (message) and move it to pkg?